### DISCUSSION TITLE
How do I get microphone access?

### DISCUSSION THREAD

**Discussion by winteriscariot** [filips123/PWAsForFirefox#133] [Q&A] [answered]
I'm using FirefoxPWA for Discord cuz I hate the electron backend. I can use the microphone in Discord in my normal firefox session, but I can't do it in the FirefoxPWA session: when I try to unmute myself, it says that I don't have a current version of Firefox. 

The Microphone Permissions dialog in Settings is not particularly helpful cuz it only lets you _remove_ permissions, not add new ones.

Is there a way to fix this?

**Comment by filips123** [comment, marked as answer]
It works fine for me on Windows and Ubuntu. Which OS do you use? Also, what is your user-agent header (you can check it if you press `F6` inside PWA and then go to `https://www.whatismybrowser.com/detect/what-is-my-user-agent`)?

Do you have any privacy-related addons or settings enabled? They might prevent microphone from working. And in case PWA Firefox actually isn't updated, you can also try to manually check for updates by going to PWA Menu -> Help -> About Firefox.

**Comment by winteriscariot** [comment]
I'm on Arch and the issue is my own shortsightedness: I'd replaced the firefoxpwa runtime with librewolf, switching back to stock firefox solved it. Thanks!
